.vscode/extensions.json

```
@@ -0,0 +1,15 @@
+{
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": [
+    "golang.go",
+    "editorconfig.editorconfig",
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "voorjaar.windicss-intellisense",
+    "johnsoncodehk.volar",
+    "johnsoncodehk.vscode-typescript-vue-plugin",
+    "redhat.vscode-yaml"
```

```suggestion
    "redhat.vscode-yaml",
    "davidanson.vscode-markdownlint"
```

https://open-vsx.org/extension/DavidAnson/vscode-markdownlint would be useful for docs editing, also for this file :wink: 

Could also be put into CI later.